config/projects-prod/iatlas-project.yaml

```diff
@@ -0,0 +1,26 @@
+template_path: tower-project.yaml
+stack_name: iatlas-project
+
+stack_tags:
+  Department: SCCE
```

Comment:
Just want to double check that "SCCE" is the correct department (I went off of the HTAN workspace). And also, should I be the owner? Should someone else?

Reply:
I asked these on: https://sagebionetworks.jira.com/browse/IBCDPE-431?focusedCommentId=165058.  If we don't get an answer by end of week: "SCCE" is fine, and I can be the owner.

Reply:
My latest commit updates the department and owner fields per James' comment on the ticket.